Polish/przygoda/jedzenie/wiedzmin/placki/igni/igni.txt

```diff
@@ -0,0 +1,2 @@
+Rzucasz zaklecie IGNI - z twoich rak strzelaja plomienie ognia, ktore kierujesz na nerda, przy okazji podpalajac meble stojace wzdluz sciany. Nerd wydaje okropny krzyk. Na jego twarzy pojawiaja sie nagle zmiany. Wydaje ci sie, ze on caly zmienia sie - jego cialo powoli rosnie i zmienia kszta�t. I nagle dociera to do ciebie - on nie jest czlowiekiem. To wampir, w dodatku z gatunku tych wy�szych...
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.